From 43f5c3cdb5e1e33ad5a585ddc0d860662864eeaa Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Tue, 26 Dec 2023 21:39:07 -0600
Subject: [PATCH] Don't build with netgroup support

Solus doesn't build the libnsl library that this requires currently
---
 Makefile       | 4 ++--
 hosts_access.c | 1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 3f44d85..edf8fa3 100644
--- a/Makefile
+++ b/Makefile
@@ -149,8 +149,8 @@ netbsd:
 
 linux:
 	@make REAL_DAEMON_DIR=$(REAL_DAEMON_DIR) STYLE=$(STYLE) \
-	LIBS="-lnsl" RANLIB=ranlib ARFLAGS=rv AUX_OBJ=weak_symbols.o \
-	NETGROUP="-DNETGROUP" TLI= EXTRA_CFLAGS="$(RPM_OPT_FLAGS)  -fPIC -DPIC -D_REENTRANT -DUSE_STRERROR -DSYS_ERRLIST_DEFINED -DBROKEN_SO_LINGER -Dss_family=__ss_family -Dss_len=__ss_len -DHAVE_WEAKSYMS" all
+	RANLIB=ranlib ARFLAGS=rv AUX_OBJ=weak_symbols.o \
+	TLI= EXTRA_CFLAGS="$(RPM_OPT_FLAGS)  -fPIC -DPIC -D_REENTRANT -DUSE_STRERROR -DSYS_ERRLIST_DEFINED -DBROKEN_SO_LINGER -Dss_family=__ss_family -Dss_len=__ss_len -DHAVE_WEAKSYMS" all
 
 linux-old:
 	@make REAL_DAEMON_DIR=$(REAL_DAEMON_DIR) STYLE=$(STYLE) \
diff --git a/hosts_access.c b/hosts_access.c
index 13ad9f9..37f85a2 100644
--- a/hosts_access.c
+++ b/hosts_access.c
@@ -34,7 +34,6 @@ static char sccsid[] = "@(#) hosts_access.c 1.21 97/02/12 02:13:22";
 #include <setjmp.h>
 #include <string.h>
 #include <stdlib.h>
-#include <rpcsvc/ypclnt.h>
 #include <netdb.h>
 
 extern char *fgets();
-- 
2.43.0

